linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_pam_unix_rounds/rule.yml

```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4
+
+title: 'Set number of Password Hashing Rounds'
+
+description: |-
+    Configure the number or rounds for the password hashing algorithm. This can be
+    accomplished by using the <tt>rounds</tt> option for the <tt>pam_unix</tt> PAM module.
+    <br /><br />
+    In the files <tt>/etc/pam.d/system-auth</tt> and <tt>/etc/pam.d/password-auth</tt>, append <tt>rounds=<sub idref="var_password_pam_unix_rounds" /></tt>
```

Comment:
There needs to be 2 rules here... one for system-auth and the other for password-auth. Not all systems configure password-auth as they don't always have network connectivity.

Reply:
Alright, I split the rule.